n2y/plugins/jinjarenderpage.py

```diff
@@ -182,7 +183,13 @@ def __init__(self, client, notion_data, page, get_children=True):
             # defaults and it could provide a mechanism to access the page
             # content from within the Jinja templates, which isn't possible
             # right now.
-            self.databases[database.title.to_plain_text()] = database_to_yaml(
+            database_name = database.title.to_plain_text()
+            if database_name in self.databases:
```

Comment:
Should this be a hard error?

Reply:
I actually agree that it should be.

Reply:
Great! I'm assuming you'll implement these changes :)